lib/namespace.js

```diff
@@ -1,22 +1,75 @@
 'use strict';
 
+var _ = require('lodash');
 var uptown = require('uptown');
 var createClass = uptown.createClass;
 
-var BaseElement;
-
 /*
- * A refract element implementation with an extensible element registry.
+ * A refract element implementation with an extensible namespace, able to
+ * load other namespaces into it.
  *
- * The element registry allows you to register your own classes to be instantiated
+ * The namespace allows you to register your own classes to be instantiated
  * when a particular refract element is encountered, and allows you to specify
  * which elements get instantiated for existing Javascript objects.
  */
 
-var ElementRegistry = createClass({
-  constructor: function() {
+var Namespace = createClass({
+  constructor: function(noDefault) {
```

Comment:
Changing this to `options` now. Seems like we may add other stuff here in the future.